### DISCUSSION TITLE
USACE Requirements for OpenDCS in CWBI:  End-User Permissions to Modify

### DISCUSSION THREAD

**Discussion by jbatterman** [opendcs/opendcs#335] [General]
The Missouri River Basin Water Management Office has brainstormed some requirements for what is needed for USACE to operate OpenDCS in the Cloud.  We are looking for feedback on these requirements, and additions from other offices who have different data acquisition processes.  There will be a new Discussion for each requirement.

End users need permission to modify DECODES, computations, groups, algorithms, etc.  If end users do not have permission, changes need to be completed within 2 hours for critical gages.  Critical gage designations could change over time based on current flooding conditions and time of year.

@MikeNeilson @hoffmanjs @opendcs/opendcs-core-devs @opendcs/opendcs-governance @opendcs/opendcs-triage

**Comment by MikeNeilson** [comment]
The current plan is to grant appropriate permissions, barring any issues. I don't think USACE could reasonably standup a group that meets that requirement and my personal disagreement with 2 hours is that it's still too long.

There will likely need to be a little work to make permissions a bit more granular, but I don't foresee any hurdles there other than volume of other work. (And waiting on the new rest_api to be ready for operation and extension/change.)
